src/build_options.rs

```diff
@@ -70,6 +70,10 @@ pub struct BuildOptions {
     #[clap(short, long, multiple_values = true, multiple_occurrences = true)]
     pub interpreter: Vec<PathBuf>,
 
+    /// Find interpreters from the host machine
+    #[clap(short = 'F', long, conflicts_with = "interpreter")]
```

Comment:
I'm not sure which short one is better, `-F` or `-I`?

Reply:
would go with `-f` (lowercase)